src/fusion.cpp

```diff
@@ -389,15 +389,18 @@ std::string LogCmdBnormFusion(const miopenFusionPlanDescriptor_t fusePlanDesc, i
 
     if(bn_op != nullptr)
     {
-        str += BnormArgsForMIOpenDriver(&bn_op->input_desc,
-                                        &bn_op->base_desc,
-                                        bn_op->mode,
-                                        nullptr,
-                                        nullptr,
-                                        nullptr,
-                                        nullptr,
-                                        miopen::debug::BatchNormDirection_t::ForwardInference,
-                                        false);
+        // str += BnormArgsForMIOpenDriver(&bn_op->input_desc,
+        //                                 &bn_op->base_desc,
+        //                                 nullptr,
+        //                                 nullptr,
+        //                                 nullptr,
+        //                                 bn_op->mode,
+        //                                 nullptr,
+        //                                 nullptr,
+        //                                 nullptr,
+        //                                 nullptr,
+        //                                 miopen::debug::BatchNormDirection_t::ForwardInference,
+        //                                 false);
```

Comment:
Should this be removed? 

Not really sure what makes sense here tbh.

Reply:
ok will remove this. It a fusion and driver command does not make sense to me.